src/components/Input.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+export default function Input({ 
+    children,
+    kind,
+    className,
+    typeInput, 
+    placeHolder
+}) {
+    return (
+        <input 
+        className={classnames(`input input-${kind}`, className)}
+        type={typeInput}
+        placeholder={placeHolder}
+        >
+            {children}
+        </input>
+    )  
+}
```

Comment:
A pior coisa que pode se fazer em desenvolvimento é dar ctrl c e ctrl v em um código. Sempre use códigos para se INSPIRAR e não para copiar literalmente igual. Pare para refletir e ver se o que vc tá fazendo faz sentido lógico para aquele código. Aqui, por exemplo, vc provavelmente copiou do componente Button, mas vc não usa `kind` pra este componente, 0 necessidade, nem mesmo tem classes diferentes no css, então qual o sentido de ter `kind` aqui?  `classname` também não é usado, não tem porque usar. E `children` pare um pouco parra pensa, a tag input do HTML tem `children`? Vai algo no "meio" do input?  Input é tag única `<input />` e não `<input></input>` 

Dê um nome melhor para a classe, input é muito genérico, seja mais específico, pensa em outro nome
```suggestion
export default function Input({ type, placeHolder }) {
    return (
        <input 
          className="CLASSE_DO_INPUT"
          type={type}
          placeholder={placeHolder}
        />
    )  
}
Input.propTypes = {
  type: PropTypes.string,
};

Input.defaultProps = {
  type: 'text',
}

```

Reply:
Mosquei, foi corpo mole meu, vou procurar melhorar isso ao máximo

Reply:
Alterei inputs